doc/news/2022-05-01-markor-v2.9.md

```diff
@@ -41,12 +41,159 @@ This search now also checks contents of files encrypted by Markor.
 
 
 ### Changelog
-**Notice:** Development changelog is always available [from GitHub](https://github.com/gsantner/markor/blob/master/CHANGELOG.md#readme). See the [history](https://github.com/gsantner/markor/compare/v2.6.0...v2.7.0) for code changes.<br/>
-
-- Reference to GitHub discussion on More page, by @gsantner
-- Add multi-selection to todo.txt dialogs, by @harshad1 @gsantner
-- In-content search support for encrypted files, #1388 by @opensource21
-- Remove alternative todo.txt naming (tags/categories), by @gsantner
+**Notice:** Development changelog is always available [from GitHub](https://github.com/gsantner/markor/blob/master/CHANGELOG.md#readme). See the [history](https://github.com/gsantner/markor/compare/v2.8.0...v2.9.0) for code changes.<br/>
+
+* [Add Markor blog posts markdown documents, by @gsantner](https://github.com/gsantner/markor/commit/ddb41a6e)
+* [Renumber list performance improvement, by @harshad1 (PR #1688)](https://github.com/gsantner/markor/commit/0142c2e1)
+* [Fix: Template is not applied when creating new file, by @tifish (PR #1669)](https://github.com/gsantner/markor/commit/7616818b)
+* [Enable filename text input filter at rename dialog, by @gsantner closes #1668](https://github.com/gsantner/markor/commit/2102dc6e)
+* [Cleaner widget logic, update widget on path change, by @harshad1 (PR #1660)](https://github.com/gsantner/markor/commit/79bcb47a)
+* [Take picture: Change filename template to usual camera format, by @gsantner, #1655](https://github.com/gsantner/markor/commit/7146fb62)
+* [Improve showing selection and related functions, by @harshad1 (PR #1658)](https://github.com/gsantner/markor/commit/0db38d5c)
+* [Improvements to show selection after search / heading etc, by @harshad1, closes #1325 (PR #1653)](https://github.com/gsantner/markor/commit/efb9f6a1)
+* [Update translations (PR #1654)](https://github.com/gsantner/markor/commit/40f39e99)
+* [Replace discussion reference from Matrix to GitHub discussions, by @gsantner](https://github.com/gsantner/markor/commit/3514491d)
+* [Filebrowser: Fix copy/move getCurrentFolder() NullPointer, by @gsantner](https://github.com/gsantner/markor/commit/7a3ffd04)
+* [Fix ArrayIndexOutOfBounds at ShortcutUtils, by @gsantner](https://github.com/gsantner/markor/commit/e746478c)
+* [Filemanager: disable search button at root directory, closes #1652, by @gsantner](https://github.com/gsantner/markor/commit/47de5795)
+* [Base functionality to handle Toolbar (title) clicks & long-clicks, by @gsantner](https://github.com/gsantner/markor/commit/6ac968c5)
+* [Update translations (PR #1649)](https://github.com/gsantner/markor/commit/016f2289)
+* [Simplify TextAction handling code, by @harshad1 (PR #1626)](https://github.com/gsantner/markor/commit/f2aa4b59)
```

Comment:
This is a purely internal change. IMO we don't need to have internal changes (except for performance improvements) in the blogpost. (IMO) we should remove this